src/test/java/com/karankumar/bookproject/backend/statistics/RatingStatisticsTest.java

```diff
@@ -58,39 +56,38 @@ public void beforeEachSetup() {
 
     @Test
     void lowestRatedBookExistsAndIsFound() {
-        assertEquals(bookWithNoRating.getTitle(), ratingStatistics.findLeastLikedBook().getTitle());
+        assertThat(ratingStatistics.findLeastLikedBook().getTitle()).isEqualTo(bookWithNoRating.getTitle());
```

Comment:
Could you please refactor this so that it adheres to our 100 character column limit (e.g. create variables for the actual and expected and pass those in)?